NetProbe/View/ViewController.mm

```diff
@@ -197,6 +218,9 @@ - (void) prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         else if (tappedButton.tag == 3){
             [vc setFileName:@"http-invalid-request-line"];
         }
+        else if (tappedButton.tag == 4){
+            [vc setFileName:@"web-connectivity"];
+        }
```

Comment:
Why here you're comparing against `4` and above against `self.ndt_testButton`? (Asking because in both cases the variable is named `tappedButton` but perhaps this comment is only out of ignorance)